replit/nixmodules
# Debugpy for Python 3.11

Why
===

Support debugger for Python 3.11. 

https://linear.app/replit/issue/DX-205/fix-debugpy-for-python-311

What changed
============

1. copied over debugpy derivations from nixpkgs and disabled tests
2. link it for python 3.11 case

Test plan
=========

1. make blank repl
2. add python 3.11 module
3. make a .py script, set breakpoints and run the debugger to see the breakpoints work

Rollout
=======

_Describe any procedures or requirements needed to roll this out safely (or check the box below)_
